src/components/NavBar/NavBar.jsx

```diff
@@ -61,7 +57,7 @@ const NavBar = () => {
 
   // Event handler for logging out of SOLID POD and removing items from localStorage
   const handleLogout = () => {
-    removeKeys();
+    localStorage.clear();
```

Comment:
Luckily for us, this `removeKeys` function is built into browsers, so we don't need to build it ourselves.
This will work fine as long as we use `localStorage` for info that lets us preserve login between sessions, which we're doing now. If we need to store info across different active logins we have a few options:

1. Store it browser-side in [indexeddb](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)
2. Store it serverside in a separate config file that we fetch on login.
3. Brainstorm a few other things too

These all have different advantages and tradeoffs.

Reply:
I had thought about a full clear before with localStorage.clear() before, but I've headed towards just clearing those generated by PASS.